client/src/components/QModal.jsx

```diff
@@ -1,16 +1,22 @@
 import React from 'react';
-import { useSelector, useDispatch } from 'react-redux';
+import { useDispatch } from 'react-redux';
+import { postQ } from './QAhelp/QArequests';
 
-function QModal({ product }) {
+function QModal({ pID }) {
   // const qModal = useSelector((state) => state.qList.mo)
   const dispatch = useDispatch();
 
   function onSubmitClickVal() {
-    return submitForm({
-      body: document.getElementById('#newQA'),
-      name: document.getElementById('#nick'),
-      email: document.getElementById('#email'),
-      product_id: product.product_id
+
+    console.log(document.getElementById('newQA').value);
+    console.log(document.getElementById('nick').value);
+    console.log(document.getElementById('email').value);
+
+    return postQ({
+      body: document.getElementById('newQA').value,
+      name: document.getElementById('nick').value,
+      email: document.getElementById('email').value,
+      product_id: pID
     });
```

Comment:
please use `form`'s `onSubmit` attribute to submit form data.

don't use `document.getElementById`, instead save form element values in state using props, or reference values in the submit event object passed to `onSubmit`

Reply:
updated